modules/common.nix

```diff
@@ -185,26 +185,45 @@ in {
 
     # Inline capacitated projects exposes capacitor interface
     inlineCapacitorPackages =
-        # Note, this does not re-expose current flake's self derivations, only re-uses its inputs
-        let self = context.self // {
-              # TODO: support sub-flakes, aka named environments
-              # outPath = context.self.outPath + "/dir";
-
-              # Fixed point operation that normally happens in call-flake.nix
-              inputs = context.self.inputs // { inherit self;};
-            } // project;
-
-            project = context.inputs.flox-floxpkgs.inputs.capacitor.lib.capacitor.capacitate.capacitate
-                    {}
-                    self.inputs
-                    (if lib.isFunction (config.inline-packages or null)
-                    then arg: (config.inline-packages arg)
-                    else _: config.inline-packages
-                    );
-
-            ## We only inject top-level packages
-            result = project.packages.${system} or {};
-      in builtins.attrValues result;
+      # Note, this does not re-expose current flake's self derivations, only re-uses its inputs
+      let
+        self =
+          context.self
+          // {
+            # TODO: support sub-flakes, aka named environments
+            # outPath = context.self.outPath + "/dir";
+
+            # Fixed point operation that normally happens in call-flake.nix
+            inputs = context.self.inputs // {inherit self;};
+          }
+          // project;
+
+        project =
+          context.inputs.flox-floxpkgs.inputs.capacitor.lib.capacitor.capacitate.capacitate
+          {}
+          self.inputs
+          (
+            if lib.isFunction (config.inline-packages or null)
+            then arg: (config.inline-packages arg)
+            else _: config.inline-packages
+          );
+      in
+        lib.mapAttrsToList (
+          name: drv: let
+            publishData =
+              floxpkgs.lib.readPackage {
+                # this is kind of meaningless
+                attrPath = ["inline-packages" "packages" system name];
```

Comment:
Not sure what we want this to be. Also I think the `inline-packages.packages` is kinda redundant - should we change that to `inline-flake.packages`? @ysndr any opinion on what we name this?

Reply:
If we embed the inline-packages into the attrPath's passthru we can have this be:
```
self#floxEnv.system.NAME.inline.packages.SYSTEM.ANOTHER-NAME
```

and it would be valid.

Reply:
Done